driver-mapping/src/main/java/com/datastax/driver/mapping/Mapper.java

```diff
@@ -160,7 +167,11 @@ public MappingManager getManager() {
      * @return a query that saves {@code entity} (based on it's defined mapping).
      */
     public Statement saveQuery(T entity) {
```

Comment:
[JAVA-1093](https://datastax-oss.atlassian.net/browse/JAVA-1093), although this could be done in this PR too if we decide its worth doing now.